src/utils/common.ts

```diff
@@ -83,6 +83,67 @@ export function getOverrideIP() {
     .trim();
 }
 
+/**
+ * Here we check that the value in the custom param can be considered to be True:
+ * 1. Boolean true value
+ * 2. Any number !== 0
+ * 3. 'yes' or 'true' string in lower / upper case
+ */
+export const hasTrueValue = (value: undefined | null | string | number | boolean): boolean => {
+  // null, undefined or empty string
```

Comment:
I think you could follow a similar pattern to this and it's a little more readable and maintainable:
https://github.com/jwplayer/ott-web-app/blob/6098e0f094ffa770c7a224b902bbf2d4d0ba8e8b/src/services/config.service.ts#L118

Reply:
It depends on the purpose of the function. My intention was to make it more universal to use not only for custom params. Custom params accept only strings now (just checked it), so the approach with `indexOf` will work.
But maybe it is better just to solve custom params problem and not make it more complicated.

Reply:
If you take in `unknown` and use `String(value)` it should work